src/modules/LanItems.rb

```diff
@@ -2535,13 +2535,18 @@ def clear_set_hostname
     # @param [String] device type
     # @return [String] available device name
     def new_type_device(type)
+      new_type_devices(type, 1).first
+    end
+
+    def new_type_devices(type, count)
       raise ArgumentError, "Valid device type expected" if type.nil? || type.empty?
+      return [] if count < 1
 
       known_devs = find_type_ifaces(type)
 
-      candidates = (0..known_devs.size).map { |c| "#{type}#{c}" }
+      candidates = (0..known_devs.size + count -1).map { |c| "#{type}#{c}" }
```

Comment:
this `-1` looks wrong as it is unary minus and not binary one. so `- 1`

Reply:
ok